full-service/src/db/transaction_log.rs

```diff
@@ -322,24 +327,38 @@ pub trait TransactionLogModel {
         conn: Conn
     ) -> Result<(), WalletDbError>;
 
-    /// Update the finalized block index to all pending transaction logs that associate with a given transaction output (txo).
+    /// Update the finalized block index to all pending transaction logs that have an output
+    /// transaction corresponding to `transaction_output_txo_id_hex`.
     /// 
     /// # Arguments
-    ///
-    ///| Name                    | Purpose                                                                      | Notes |
-    ///|-------------------------|------------------------------------------------------------------------------|-------|
-    ///| `txo_id_hex`            | The txo ID for which to get all transaction logs associated with this txo ID.|       |
-    ///| `finalized_block_index` | The block index at which the transaction will be completed and finalized.    |       |
-    ///| `conn`                  | An reference to the pool connection of wallet database                       |       |
-    ///
-    /// # Returns
-    /// * unit
+    /// * `transaction_output_txo_id_hex` - The txo ID for which to get all transaction logs
+    ///   associated with this txo ID.
+    /// * `finalized_block_index` - The block index at which the transaction will be completed and
+    ///   finalized.
+    /// * `conn` - A reference to the pool connection of wallet database
     fn update_pending_associated_with_txo_to_succeeded(
-        txo_id_hex: &str,
+        transaction_output_txo_id_hex: &str,
```

Comment:
noticed when I went to add the new method, I failed to update the trait docs for the previous PR.